src/main/java/com/databasepreservation/modules/siard/path/PathStrategy.java

```diff
@@ -0,0 +1,88 @@
+package com.databasepreservation.modules.siard.path;
+
+/**
+ * Interface to describe paths to folders and files for some SIARD archive.
+ *
+ * Paths to folders should end with a slash and are relative to the root of the SIARD archive
+ * Paths to files should NOT end with a slash and are relative to the root of the SIARD archive
+ * Namespaces should have a prefix and some parameters to build a URL
+ * Names are names of folders or files (for files it should also contain the extension)
+ *
+ * @author Bruno Ferreira <bferreira@keep.pt>
+ */
+public interface PathStrategy {
+	/**
+	 * Returns the path to the metadata.xml file
+	 */
+	public String metadataXmlFile();
```

Comment:
These names should be more descriptive. Example: getMetadataXmlFilePath()
Also, what are the selected path separators? Will this work on Windows?